src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2499,7 +2499,7 @@
     <value>Clear selected items</value>
   </data>
   <data name="ToggleSelectDescription" xml:space="preserve">
-    <value>Toggle item selection</value>
+    <value>Invert selection of items</value>
```

Comment:
There is a different action for inverting selection, this action is for toggling the selection of the focused item. Please revert.